sample/src/main/kotlin/com/slack/circuit/sample/MainActivity.kt

```diff
@@ -19,6 +19,8 @@ import android.app.Activity
 import android.os.Bundle
 import androidx.activity.compose.setContent
 import androidx.appcompat.app.AppCompatActivity
+import androidx.compose.material3.MaterialTheme
```

Comment:
how difficult would it be to create our own theme? just curious, i can also look into it.